www/source/images/habitat-mark.svg

```diff
@@ -1,19 +1,12 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Per our chat, let's just drop these and remove the references to them.

Reply:
Will do!